### PR TITLE
Fixes bug #1283271 - updated /foundation page content

### PR DESCRIPTION
**Pull request by mmmavis**
## Description
Updated content on /foundation page. Request came from MoFo's plan repo: https://github.com/MozillaFoundation/plan/issues/819. Copy is based on hhttps://docs.google.com/document/d/1Gu7OrQRS_q_YV4cWhat2VKGq1zRvVTaBjFzeqLsRo6k/edit 😃 

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1283271

## Checklist
- [ ] Requires l10n changes.
- [ ] Related functional & integration tests passing.

### CODE REVIEW

File: bedrock/foundation/templates/foundation/index.html [hunk 1]
```diff
@@ -13,89 +13,128 @@ <h1 class="page-title">{{ _('The Mozilla Foundation') }}</h1>
...
-      like-minded allies to bring an open source ethos to industries and
-      technologies like online video, telecommunications and virtual reality.
+    {% trans %}
+      Working with people and communities all across Mozilla, the Foundation team is focused on fueling the open internet movement. Our programs are focused in three areas:
     {% endtrans %}
   </p>
 
+  <ol>
+    <li>
+      <p><strong>{{ _('Shape the agenda.') }}</strong></p>
```
Comment (craigcook) [inline bedrock/foundation/templates/foundation/index.html]: These should probably `h3` elements

File: bedrock/foundation/templates/foundation/index.html [hunk 2]
```diff
@@ -13,89 +13,128 @@ <h1 class="page-title">{{ _('The Mozilla Foundation') }}</h1>
...
-      include our flagship <a href="{{ firefox }}">Firefox browser</a> and
-      other open source tools. The Mozilla Corporation also partners with
-      like-minded allies to bring an open source ethos to industries and
-      technologies like online video, telecommunications and virtual reality.
+    {% trans %}
+      Working with people and communities all across Mozilla, the Foundation team is focused on fueling the open internet movement. Our programs are focused in three areas:
     {% endtrans %}
   </p>
 
+  <ol>
```
Comment (craigcook) [inline bedrock/foundation/templates/foundation/index.html]: Now that I think about it, maybe an ordered list isn't really best for this copy, just putting numbers in the headings might be better: `<h3>{{ _('1. Shape the agenda') }}</h3>` ... what do you think?

File: media/css/foundation/foundation.less [hunk 3]
```diff
@@ -56,6 +56,23 @@ img {
...
+    list-style-type: disc;
+  }
+}
+
+ul + p {
+  margin-top: @baseLine;
+}
+
+.example-link {
+  &:after {
```
Comment (craigcook) [inline media/css/foundation/foundation.less]: We have a standard `.trailing-arrow` mixin for this, if that makes things easier.

**Comment by craigcook** [comment]
Since @mmmavis is on PTO today and we need to get this to prod, I made the above minor tweaks and merged this in https://github.com/mozilla/bedrock/commit/5092c4f8934eb56f73ed64b854dd48fdf1ec2cd1

**Comment by mmmavis** [comment]
@craigcook, thanks for helping me add in those tweaks ... which allowed me to immerse myself in nature longer 😎 

mavis-brandywinefallspark

**Pull request closed without merging**